docs/contributing/testing.md

```diff
@@ -0,0 +1,152 @@
+---
+id: testing
+title: Contributing
+sidebar_label: Testing
+---
+
+# Testing
+
+When building Web sites and apps on FAST-DNA it's recommended to perform exhaustive testing to instill the confidence that you've met your customers' expectations. The checklist below was compiled over the last few years and is a good starting point for manual testing Web applications.
+
+## Checklist
+
+1. Test each page for design and content issues
+    1. Test screen sizes from viewport 1 to 6
+    2. Test screen orientations
+       1. Portrait
+       2. Landscape
+    3. Test word length and case
+        1. Test smallest amount of text
+        2. Test largest amount of text
+        3. Case sensitivity
+2. Test for localization (RTL, LTR)
+    1. English
+    2. Arabic
```

Comment:
from the other PR:
@nicholasrice
nicholasrice 14 hours ago  Member
I think it is disingenuous to advise testing languages when we don't have infrastructure for doing it - should we remove language testing until we facilitate that with our product?

  @awentzel
awentzel 4 hours ago  Member
We're sharing what we believe to be important to test. This doesn't mean we have to provide a means of doing all of this. If you build a website with FAST-DNA then it's up to you to wire up localization not us. Which is why I removed the localization documentation as well. But, I think it's useful to share what we believe to be valid testing for quality.

  @nicholasrice
nicholasrice 3 hours ago  Member
That doesn't seem right to me. The context of this markdown file is FAST-DNA. I wouldn't come here expecting to see general guidance of what we advise consumers to do in regards to testing - thats none of our concern. I would expect to come to this file to learn about testing requirements for the FAST-DNA, which currently does not include locale testing.

Reply:
I believe that providing guidance is our concern because it adds credibility.  I also, believe that any questions that are asked of us should include documentation with an answer.  This was written because people were asking about it.  If we want to move this and our other stuff under a  "Guidance" section that just relates to how we do work I'm ok with that.  I would love it if all these testing sites for example would provide guidance.  None do.  It's frustrating.   If we have knowledge we should share it.  @Falkicon to get his thoughts.  But, I don't think this is a blocking matter.

Reply:
Yea - I just think there are a million and a half resources on testing and that each product is going to have different requirements for testing. To me, the value-add is to speak to how *this project* tests, not how a project we know nothing about should test.

If we want a write-up on how we feel UI should be comprehensively tested that is one thing, but this is document about *contributing* and should be scoped accordingly.

Reply:
after talking to Nick, removed the localization for regions since we don't have the means